cmd/ore/azure/replicate-image.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build go1.7
+
+package azure
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdReplicateImage = &cobra.Command{
+		Use:   "replicate-image image offer sku version [regions...]",
```

Comment:
With this many positional parameters I would favor making them required options instead. And regions could be a repeatable --region option. Then you can offer more help text on what the heck an "offer" or an "sku" is because I have no idea.